packages/gatsby-theme/src/components/Image/index.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+/** @jsx jsx */
```

Comment:
I think this pragma **/** @jsx jsx */** is not necessary for Image component.

Reply:
Ready, set, go!
:dancers: